InsetLabel/InsetLabel.swift

```diff
@@ -79,6 +79,11 @@ open class InsetLabel: UILabel {
         let newRect = UIEdgeInsetsInsetRect(rect, contentInsets)
         super.drawText(in: newRect)
     }
+    
```

Comment:
Could you add the MARK line?

```
// MARK: UIView (UIViewHierarchy)

```

Reply:
Thank you for your pull request!

Could you change the target from `GyazSquare:master` to `GyazSquare:develop`?

I'll merge it and release it!

Thanks in advance.